src/devicemanager/devicemanager.h

```diff
@@ -175,6 +176,7 @@ class DeviceManager : public QAbstractListModel
 
     QSharedPointer<Sensor> _primarySensor;
     ProtocolDetector* _detector;
+    Ping360EthernetFinder* _ping360EthernetFinder;
```

Comment:
Is there a need for this to be a pointer ?

Reply:
nope, fixed.